package.json

```diff
@@ -7,7 +7,7 @@
     "build": "lerna run build",
     "test": "lerna run test",
     "test:debug": "lerna run test:debug",
-    "test:integration": "lerna run test:integration",
+    "test:integration": "npm run test --prefix integration_tests",
```

Comment:
I assume that we have to use npm here instead of lerna because it's not in the packages directory. Is that correct?

Reply:
yup

Reply:
Correct